proto/process/agent.proto

```diff
@@ -651,11 +651,24 @@ enum EphemeralPortState {
 	ephemeralFalse = 2;
 }
 
+message ProtocolInfo {
+	ProtocolType type = 1;
+
+	// Additional tags relevant for the protocol.
+	// Examples - tls_version:1.3, http_version:2.
+	repeated string tags = 2;
```

Comment:
This format is quite complex (cardinality, memory) and should be a dictionary in the connections like for the connection Tags.

I think we don't need a new format, but just adding tags to the connection, the backend support it already and the frontend will.
We could add protocols_path:tcp.tls.http and tls_openssl:1.3 http:2.0 to the connection

https://github.com/DataDog/agent-payload/pull/183/files#diff-69b34851fbf6263111121260c46c57084a1fe6e34e9b750a5a673694f2dfe1b0L719-L723

Reply:
@nplanel I simplified the message to contain a list of protocols
left most is the bottom most protocol, and right most is the top most protocol

Reply:
I removed the tags at the moment
do you think I should restore it?